source/installation/resource-manager/sge.rst

```diff
@@ -93,3 +93,20 @@ This sets the sh shell to behave like bash and ensures you've sourced your users
             set +o posix
             . ~/.bashrc
             %s
+
+Invalid Job name
+****************
+You're likely to encounter an issue in running batch connect applications complaining about invalid
```

Comment:
>You're likely to encounter an issue in...

change to

>If you encounter an issue with invalid job names when running batch connect like the error below:

The truth is if you are using SGE you are not likely to experience this. Many sites have been running OnDemand with SGE for a while now and we only recently heard of this issue with illegal job name, so it only affects a small subset of the SGE sites. That may be due to the SGE flavor UGE being more forgiving.